web/src/lib/data-hooks/useUserData.test.tsx

```diff
@@ -246,4 +246,55 @@ describe("useUserData", () => {
       expect(mockApi.postUserData).not.toHaveBeenCalled();
     });
   });
+
```

Comment:
I need help figuring oud how to test the new functionality. The commented code is no working and will need to be removed before merging code into main.